packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/dataStreamSearch/workspaceSelector/workspaceSelector.tsx

```diff
@@ -33,6 +38,7 @@ export function WorkspaceSelector({ control, client }: WorkspaceSelectorProps) {
           errorText={fieldState.error?.message}
         >
           <Select
+            disabled={status == 'error'}
```

Comment:
the screenshot does not have a error state within the dropdown right ?

Reply:
then can you remove it ? the screenshot does not seem to have it

Reply:
The ss is not showing open state of dropdown, not sure that we need to remove it